src/components/VoteButton.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react"
+import { patchUpVote } from "../utils/api"
+import '../.css/VoteButton.css'
+
+const VoteButton = ({article, setArticle}) => {
+    const [voteClicked, setVoteClicked] = useState(true)
+
+    const handleUpVotes = async (event) => {
+        event.preventDefault()
+        setVoteClicked(!voteClicked)
+        if (voteClicked) {
+            const updatedArticle = {...article, votes: article.votes +1}
+            setArticle(updatedArticle)
+            patchUpVote(article.article_id, {inc_votes: 1})
+            .then((response) => {
+                console.log(response.data.article, "upvote")
+            })
+            .catch((err) => console.log(err))
+        } else {
+            const updatedArticle = {...article, votes: article.votes -1}
+            setArticle(updatedArticle)
+            patchUpVote(article.article_id, {inc_votes: -1})
+            .then((response) => {
+                console.log(response.data.article, "downvote")
+            })
+            .catch((err) => console.log(err))
+        }
```

Comment:
There is a lot of duplicate code here, how can this be refactored so you only have one request here but what you are providing it changes depending on voteClicked? 

This is also not dealing with the error if there ever is an error we receive.